### PR TITLE
add tag info to buildparameter output

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@mfojtik ptal.

**Comment by bparees** [comment]
[test]

File: pkg/cmd/cli/describe/describer.go [hunk 1]
```diff
@@ -160,10 +160,14 @@ func describeBuildParameters(p buildapi.BuildParameters, out *tabwriter.Writer)
 		}
 	}
 	if p.Output.To != nil {
+		tag := buildapi.DefaultImageTag
+		if p.Output.Tag != "" {
```
Comment (smarterclayton) [inline pkg/cmd/cli/describe/describer.go]: `len(p.Output.Tag)` (convention)

**Comment by bparees** [comment]
@smarterclayton convention conformed to.

**Comment by smarterclayton** [comment]
Conformity noted

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**